### PR TITLE
Add Russ Perry

### PR DESCRIPTION
**Pull request by rperry99**
Hello! I added my name to the data.json file. Let me know if there are any issues!

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -136,5 +136,11 @@
...
         "imageUrl": "https://user-images.githubusercontent.com/45217974/91754997-ae650a80-ebca-11ea-97fd-dfa011b49d79.png",
         "issueNumber": 72
-    }
+    }, 
+    {
+        "name": "Russ Perry",
+        "githubUsername": "rperry99",
+        "imageUrl": "https://user-images.githubusercontent.com/24259120/92329693-1a19fe00-f037-11ea-889a-8fb530b03514.png",
+        "issueNumber": 78
+    },
```
Comment (eddiejaoude) [inline data.json:145]: I think the trailing `,` will break `json`

**Comment by eddiejaoude** [comment]
Looks good Russ, just 1 small change request - I made it inline, hopefully it is clear, any questions let me know 🤓

**Comment by rperry99** [comment]
Oops, my bad! I am used to putting the hanging commas in my code! I think I submitted the change? 😂

**Comment by eddiejaoude** [comment]
We do the trailing comma in typescript too, we all do it, silly json 🤓

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**